src/renderer/wg_engine/tvgWgCommon.cpp

```diff
@@ -76,7 +79,9 @@ void WgContext::initialize(WGPUInstance instance, WGPUSurface surface)
     };
     // request device
     wgpuAdapterRequestDevice(adapter, &deviceDesc, onDeviceRequestEnded, &device);
-    assert(device);
```

Comment:
Isn't it required by wgpu-native? I assume wgpu-native need to ensure whether device is valid.
We can put assertion after `emscripten_sleep` ilne, it would work in web as well.

Reply:
we can keep it.
fixed